data/tutorials/gs_00_up_and_running.md

```diff
@@ -53,9 +50,15 @@ below](#up-and-running-with-the-platform-installer) for the instructions.
 However, please note that it is still experimental and in active development
 (any feedback is highly appreciated).
 
-### Installation for Unix
+### Installation on Unix (Linux, macOS, *BSD)
```

Comment:
This breaks the link `[Installation for Unix, including Linux and macOS](#installation-for-unix)`